recipes/kfserving.rb

```diff
@@ -0,0 +1,208 @@
+# Install KFServing and required tools
+
+# Istio
+
+directory "/home/#{node['kube-hops']['user']}/istio" do
+  owner node['kube-hops']['user']
+  group node['kube-hops']['group']
+  mode '0700'
+  action :create
+end
+
+bash 'install-istioctl' do
+  user node['kube-hops']['user']
+  group node['kube-hops']['group']
+  environment ({ 'HOME' => ::Dir.home(node['kube-hops']['user']) })
+  cwd ::Dir.home(node['kube-hops']['user'])
+  code <<-EOH
+    cd istio
+    istio_tar="istio-#{node['kube-hops']['istio_version']}-linux-amd64.tar.gz"
+    wget -nc #{node['kube-hops']['istio_url']}
+    tar zxf $istio_tar
+    rm -f $istio_tar
+    rm -f istio
+    ln -s istio-#{node['kube-hops']['istio_version']} istio
+    EOH
+  not_if { File.exist? "/home/#{node['kube-hops']['user']}/istio/istio-#{node['kube-hops']['istio_version']}/bin/istioctl" }
+end
+
+template "/home/#{node['kube-hops']['user']}/istio/istio-operator.yaml" do
+  source "istio-operator.yml.erb"
+  owner node['kube-hops']['user']
+  group node['kube-hops']['group']
+end
+
+bash 'apply-istio' do
+  user node['kube-hops']['user']
+  group node['kube-hops']['group']
+  environment ({ 'HOME' => ::Dir.home(node['kube-hops']['user']) })
+  cwd ::Dir.home(node['kube-hops']['user'])
+  code <<-EOH
+    cd istio
```

Comment:
`cwd` above (line 39) allows you to specify the current working directory, add `istio` there instead of doing `cd` later.

Reply:
Done